shared/menubar/remote-serializer.desktop.tsx

```diff
@@ -67,9 +67,9 @@ export const serialize: any = {
   kbfsDaemonStatus: (v: FSTypes.KbfsDaemonStatus) => v,
   kbfsEnabled: (v: boolean) => v,
   loggedIn: (v: boolean) => v,
-  outOfDate: (v: boolean) => v,
```

Comment:
I'm not sure why this was ever typed as a boolean, we were definitely sending an object. Updated the type below.